make pytest fails

please let me know if there's a forum or mailing list where it would be better to submit.

I d/l pybind11 and successfully ran cmake (using gcc-4.9.2, Python 2.7.10)
Unf, 'make pytest' fails; can you offer any advice? Thanks!

[  3%] Building CXX object tests/CMakeFiles/pybind11_tests.dir/test_sequences_and_iterators.cpp.o
/p/lscratche/hysom/pybind11/tests/test_sequences_and_iterators.cpp:132:68: error: converting to 'std::unordered_map<std::basic_string<char>, std::basic_string<char> >' from initializer list would use explicit constructor 'std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::unordered_map(std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::size_type, const hasher&, const key_equal&, const allocator_type&) [with _Key = std::basic_string<char>; _Tp = std::basic_string<char>; _Hash = std::hash<std::basic_string<char> >; _Pred = std::equal_to<std::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::basic_string<char>, std::basic_string<char> > >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::size_type = long unsigned int; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::hasher = std::hash<std::basic_string<char> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::key_equal = std::equal_to<std::basic_string<char> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::allocator_type = std::allocator<std::pair<const std::basic_string<char>, std::basic_string<char> > >]'
     StringMap(std::unordered_map<std::string, std::string> init = {})
                                                                    ^
In file included from /p/lscratche/hysom/pybind11/tests/pybind11_tests.h:2:0,
                 from /p/lscratche/hysom/pybind11/tests/test_sequences_and_iterators.cpp:11:
/p/lscratche/hysom/pybind11/include/pybind11/pybind11.h: In instantiation of 'pybind11::detail::init<Args>::execute(Class&, const Extra& ...)::<lambda(Base*, Args ...)> [with Class = pybind11::class_<StringMap>; Extra = {}; typename std::enable_if<(! Class:: has_alias), int>::type <anonymous> = 0; Args = {}; Base = StringMap]':
/p/lscratche/hysom/pybind11/include/pybind11/pybind11.h:1246:29:   required from 'struct pybind11::detail::init<Args>::execute(Class&, const Extra& ...) [with Class = pybind11::class_<StringMap>; Extra = {}; typename std::enable_if<(! Class:: has_alias), int>::type <anonymous> = 0; Args = {}]::<lambda(using Base = using type = class StringMap*)>'
/p/lscratche/hysom/pybind11/include/pybind11/pybind11.h:1246:9:   required from 'static void pybind11::detail::init<Args>::execute(Class&, const Extra& ...) [with Class = pybind11::class_<StringMap>; Extra = {}; typename std::enable_if<(! Class:: has_alias), int>::type <anonymous> = 0; Args = {}]'
/p/lscratche/hysom/pybind11/include/pybind11/pybind11.h:1005:9:   required from 'pybind11::class_<type_, options>& pybind11::class_<type_, options>::def(const pybind11::detail::init<Args ...>&, const Extra& ...) [with Args = {}; Extra = {}; type_ = StringMap; options = {}]'
/p/lscratche/hysom/pybind11/tests/test_sequences_and_iterators.cpp:222:26:   required from here
/p/lscratche/hysom/pybind11/include/pybind11/pybind11.h:1246:60: error: converting to 'std::unordered_map<std::basic_string<char>, std::basic_string<char> >' from initializer list would use explicit constructor 'std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::unordered_map(std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::size_type, const hasher&, const key_equal&, const allocator_type&) [with _Key = std::basic_string<char>; _Tp = std::basic_string<char>; _Hash = std::hash<std::basic_string<char> >; _Pred = std::equal_to<std::basic_string<char> >; _Alloc = std::allocator<std::pair<const std::basic_string<char>, std::basic_string<char> > >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::size_type = long unsigned int; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::hasher = std::hash<std::basic_string<char> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::key_equal = std::equal_to<std::basic_string<char> >; std::unordered_map<_Key, _Tp, _Hash, _Pred, _Alloc>::allocator_type = std::allocator<std::pair<const std::basic_string<char>, std::basic_string<char> > >]'
         cl.def("__init__", [](Base *self_, Args... args) { new (self_) Base(args...); }, extra...);
                                                            ^
/p/lscratche/hysom/pybind11/include/pybind11/pybind11.h:976:13: error: 'pybind11::class_<type_, options>& pybind11::class_<type_, options>::def(const char*, Func&&, const Extra& ...) [with Func = pybind11::detail::init<Args>::execute(Class&, const Extra& ...) [with Class = pybind11::class_<StringMap>; Extra = {}; typename std::enable_if<(! Class:: has_alias), int>::type <anonymous> = 0; Args = {}]::<lambda(Base*)>; Extra = {}; type_ = StringMap; options = {}]', declared using local type 'pybind11::detail::init<Args>::execute(Class&, const Extra& ...) [with Class = pybind11::class_<StringMap>; Extra = {}; typename std::enable_if<(! Class:: has_alias), int>::type <anonymous> = 0; Args = {}]::<lambda(Base*)>', is used but never defined [-fpermissive]
     class_ &def(const char *name_, Func&& f, const Extra&... extra) {
             ^
make[3]: *** [tests/CMakeFiles/pybind11_tests.dir/test_sequences_and_iterators.cpp.o] Error 1

--- a/tests/test_sequences_and_iterators.cpp
+++ b/tests/test_sequences_and_iterators.cpp
@@ -129,7 +129,8 @@ private:
 // map-like functionality.
 class StringMap {
 public:
-    StringMap(std::unordered_map<std::string, std::string> init = {})
+    StringMap() = default;
+    StringMap(std::unordered_map<std::string, std::string> init)
         : map(std::move(init)) {}
 
     void set(std::string key, std::string val) {

